modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -42,6 +45,11 @@ spec:
 <5> The name of the Linux bridge configured on the node.
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 <7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.
++
+[NOTE]
+====
+A Linux bridge network attachment definition is the most efficient method for configuring a VLAN interface and provides the most control.
+====
 
 . Create the network attachment definition:
 +
```

Comment:
Small nit about the variable "<a-bridge-network>". Does the "a" mean anything? I know it's in the NAD manifest but it looks weird to me. This comment is probably out of scope.

Reply:
Yes, I axed "a" --- that definitely makes a user look twice at something that should be very plainly worded.

Changed the Verification step to:

.Verification

* Verify that the network attachment definition was created:
** Run `oc get network-attachment-definition <name>`. For example:
+
[source,terminal]
----
$ oc get network-attachment-definition <bridge-network>
----